src/churn/churn.ts

```diff
@@ -144,6 +144,10 @@ export var filterCandidatesFromSdp = (sdp:string) : string => {
   //         https://github.com/uProxy/uproxy/issues/585
   export class Connection implements peerconnection.PeerConnection<ChurnSignallingMessage> {
 
+    // Maximum time to spend gathering ICE candidates:
+    //  https://github.com/uProxy/uproxy/issues/1795
+    private static PROBE_TIMEOUT_MS = 3000;
```

Comment:
Trailing _ for private constants.  Also, if the real goal here is to avoid making the user wait too long, that might be worth documenting in a comment.